src/__tests__/Input.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { render, screen, fireEvent } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import "@testing-library/jest-dom"
+import Input from "../components/Input"
+import React from "react"
+
+describe("Input component", () => {
+  beforeEach(() => {
+    render(<Input />)
+  })
+  it("should display form on the main page", () => {
+    const form = screen.getByTestId("form")
+    expect(form).toBeVisible()
+  })
+  it("should display submit button", () => {
+    const button = screen.getByText("submit")
+    expect(button).toBeInTheDocument()
+  })
+  it("should display skip button", () => {
+    const button = screen.getByRole('button', {name: 'skip'})
+    expect(button).toBeInTheDocument()
+  })
+  it("should clear input field after clicking submit button", () => {
+    const inputElement = screen.getByTestId("input-field")
+    const button = screen.getByRole("button", {name: 'submit'})
+    userEvent.click(button)
+
+    expect(inputElement).toHaveTextContent("Enter your RegEx here")
+  })
+  it("should have className 'text-field", () => {
```

Comment:
it is not necessary to test class existance if some class exists when from the beginning